src/rollback_to_stable/rts_btree.c

```diff
@@ -321,9 +321,9 @@ __rts_btree_ondisk_fixup_key(WT_SESSION_IMPL *session, WT_REF *ref, WT_ROW *rip,
      * have moved this value to the history store as a full value. Therefore, we can safely ignore
      * the on page value if it is overflow removed.
      */
-    if (__wt_cell_type_raw(unpack->cell) == WT_CELL_VALUE_OVFL_RM)
+    if (__wt_cell_type_raw(unpack->cell) == WT_CELL_VALUE_OVFL_RM) {
```

Comment:
Why added braces?

Reply:
That was picked up by one of the code analysers.

Reply:
I thought that came from a code analyser, but actually it was needed for a temporary change that I've now removed. I'll remove the braces.

Reply:
Now removed.